modules/lazyload/module.php

```diff
@@ -96,12 +107,20 @@ public function __construct() {
 		}
 
 		add_action( 'elementor/element/after_add_attributes', function( Element_Base $element ) {
+			if ( ! $this->is_document_support_lazyload() ) {
+				return;
+			}
 			$this->update_element_attributes( $element );
 		} );
 
-		add_filter('elementor/files/css/selectors', function( $control, $value ) {
-			return $this->append_lazyload_selector( $control, $value );
-		}, 10, 2 );
+		add_action( 'elementor/frontend/element/before_enqueue_styles', function( Element_Base $element ) {
+			if ( ! $this->is_document_support_lazyload() ) {
+				return;
+			}
+			add_filter('elementor/files/css/selectors', function( $control, $value ) {
+				return $this->append_lazyload_selector( $control, $value );
+			}, 10, 2 );
```

Comment:
Can you try avoiding using `filter` inside another `action`?

Why not use `is_document_support_lazyload` inside `elementor/files/css/selectors`?

Reply:
This code doesn't work because there is no document yet at that moment. 
the document is null.

Reply:
Update : 
I seceded to remove the new action., And use different approach.